test/test_autograd.py

```diff
@@ -3204,6 +3204,361 @@ def backward(ctx, gO):
                     out.backward()
             self.assertIn('MyFunc.apply', str(w[0].message))
 
+    @unittest.skipIf(not torch.cuda.is_available(), "CUDA unavailable")
```

Comment:
Is there not a CPU version of this UX to test? In fact, you may want to create a device generic test.

Reply:
(This suggestion mooted by torch.cuda.amp change.)